src/components/PostItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {FC} from "react";
+import {IPostInList} from "../types/types";
+import {Box, Button, Card, CardActions, CardContent, Typography,} from "@mui/material";
```

Comment:
запятая последняя выглядит опасно)
```suggestion
import {Box, Button, Card, CardActions, CardContent, Typography} from "@mui/material";
```